Fabric.Identity.API/Services/LinuxCertificateService.cs

```diff
@@ -40,6 +41,12 @@ public X509Certificate2 GetEncryptionCertificate(SigningCertificateSettings cert
             throw new FabricConfigurationException("Do not encrypt settings when running on a Linux container, instead use Docker Secrets to protect sensitive configuration settings.");
         }
 
```

Comment:
I noticed this implementation is the same as the WindowsCertificateService. Will this always be the case? If so, maybe make this virtual in a base class as the default implementation?

Reply:
@michaelvidal24, not sure my suggestion is worth it for 2 lines of code?

Reply:
I had this exact debate before I pushed up the branch, :-). My initial thought was that the overhead of an abstract base class outweighed the benefit, since this is the only small method with overlap in implementation.

Reply:
Yeah, after thinking about it, I think I would just leave it as is also.